torchtext/transforms.py

```diff
@@ -597,6 +597,23 @@ def _encode(self, text: str) -> List[str]:
         tokens_ids_str: List[str] = [str(token_id) for token_id in token_ids]
         return tokens_ids_str
 
+    @torch.jit.export
+    def _batch_encode(self, text: List[str]) -> List[List[str]]:
+        """Encode text into a list of tokens IDs
+
+        Args:
+            text: An input text string.
+
+        Returns:
+            A list of token ids represents each sub-word
```

Comment:
Can we update the docstring to reflect that we're operating on a list of strings and the output is a nested list of tokens